modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/database/H2Tree.java

```diff
@@ -148,7 +167,9 @@ protected H2Tree(
             if (metaInfo.useUnwrappedPk())
                 throw new IgniteCheckedException("Unwrapped PK is not supported by current version");
 
-            this.inlineSize = metaInfo.inlineSize();
+            inlineSize = metaInfo.inlineSize();
```

Comment:
For my understanding. How is it possible that `inlineSize` passed to a constructor is not equal to `metaInfo.inlineSize()`?

Reply:
Where do we calculate it?

Reply:
@pavlukhin see org.apache.ignite.internal.processors.query.h2.database.H2TreeIndex#computeInlineSize